.worktrees/backport-release_3_6/lizmap/modules/admin/zones/project_list.zone.php

```diff
@@ -0,0 +1,271 @@
+<?php
```

Comment:
You are trying to master git with worktrees ;-)

Reply:
not anymore